<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="addUser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>106</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>添加用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewUser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>106</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>查看用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeUser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>106</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>删除用户</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>171</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>301</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>106</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
